optaweb-vehicle-routing-tests/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2021 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>optaweb-vehicle-routing</artifactId>
+    <groupId>org.optaweb.vehiclerouting</groupId>
+    <version>8.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>optaweb-vehicle-routing-tests</artifactId>
+
+  <name>OptaWeb Vehicle Routing end-to-end Tests</name>
+
+  <properties>
+    <application.host>localhost</application.host>
+    <application.port>8180</application.port>
+    <application.url>http://${application.host}:${application.port}</application.url>
+    <!-- Override to run with Podman. -->
+    <container.runtime>docker</container.runtime>
+    <frontend.project.name>optaweb-vehicle-routing-frontend</frontend.project.name>
+    <java.module.name>org.optaweb.vehiclerouting</java.module.name>
+    <version.cypress.docker>4.12.1</version.cypress.docker>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.optaweb.vehiclerouting</groupId>
+      <artifactId>optaweb-vehicle-routing-standalone</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-h2</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-maven-plugin</artifactId>
+        <version>${version.io.quarkus}</version>
+        <extensions>true</extensions>
+        <executions>
+          <execution>
+            <goals>
+              <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>integration-tests</id>
+      <activation>
+        <property>
+          <name>integration-tests</name>
+          <value>true</value>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.soulwing</groupId>
```

Comment:
What this for? to generate a unique directory for a database? Why not use an in-memory database for tests (ex: `jdbc:h2:mem:vehiclerouting;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false`)

Reply:
The idea was to have the database in a file so that we have something to inspect in case the test fails. The unique name was there to guarantee test isolation so even if you run `mvn verify` without `clean` the test will run on an empty DB.

Thanks for pointing this out. You made me think about it and realize this is unnecessary complexity providing a feature that's worthless. If the potential bug can is reproducible, one can always switch to a file based database manually.